core/src/test/resources/test_tumor_type_summaries.tsv

```diff
@@ -158,19 +158,19 @@ ERBB2	fusion	Breast cancer	ERBB2, a receptor tyrosine kinase, is altered by muta
-FGFR2	GFR2-MARVELD3 fusion	Endometrial Cancer	FGFR2, a receptor tyrosine kinase, is altered by mutation, chromosomal rearrangement or amplification in various cancer types.	The GFR2-MARVELD3 fusion is likely oncogenic.	While the pan-FGFR-targeted inhibitor erdafitinib is FDA-approved for the treatment of patients with FGFR2 or FGFR3 mutant urothelial cancer and the pan-FGFR-targeted inhibitors pemigatinib and futibatinib are FDA-approved for the treatment of patients with FGFR2-fusion positive cholangiocarcinoma, their clinical utility in patients with FGFR2 fusion-positive endometrial cancer is unknown.
-FGFR2	KIF14-FGFR2 fusion	Cancer of Unknown Primary	FGFR2, a receptor tyrosine kinase, is altered by mutation, chromosomal rearrangement or amplification in various cancer types.	The KIF14-FGFR2 fusion is likely oncogenic.	While the pan-FGFR-targeted inhibitor erdafitinib is FDA-approved for the treatment of patients with FGFR2 or FGFR3 mutant urothelial cancer and the pan-FGFR-targeted inhibitors pemigatinib and futibatinib are FDA-approved for the treatment of patients with FGFR2-fusion positive cholangiocarcinoma, their clinical utility in patients with FGFR2 fusion-positive cancer of unknown primary is unknown.
+FGFR2	GFR2-MARVELD3 fusion	Endometrial Cancer	FGFR2, a receptor tyrosine kinase, is altered by mutation, chromosomal rearrangement or amplification in various cancer types.	The GFR2-MARVELD3 fusion is likely oncogenic.	While the pan-FGFR-targeted inhibitors pemigatinib and futibatinib are FDA-approved for the treatment of patients with FGFR2-fusion positive cholangiocarcinoma, and erdafitnib is FDA-approved for the treatment of patients with urothelial cancer with susceptible FGFR genetic alterations, their clinical utility in patients with GFR2-MARVELD3 fusion positive endometrial cancer is unknown.
+FGFR2	KIF14-FGFR2 fusion	Cancer of Unknown Primary	FGFR2, a receptor tyrosine kinase, is altered by mutation, chromosomal rearrangement or amplification in various cancer types.	The KIF14-FGFR2 fusion is likely oncogenic.	While the pan-FGFR-targeted inhibitors pemigatinib and futibatinib are FDA-approved for the treatment of patients with FGFR2-fusion positive cholangiocarcinoma, and erdafitnib is FDA-approved for the treatment of patients with urothelial cancer with susceptible FGFR genetic alterations, their clinical utility in patients with KIF14-FGFR2 fusion positive cancer of unknown primary is unknown.
 FGFR3	K650E	Glioma	FGFR3, a receptor tyrosine kinase, is altered by mutation, chromosomal rearrangement or amplification in various cancers, most frequently in bladder cancer.	The FGFR3 K650E mutation is known to be oncogenic.	While there is promising clinical data in patients with urothelial cancer harboring functionally characterized hotspot FGFR3 mutations treated with pan-FGFR-targeted inhibitors such as AZD4547, BGJ398, Debio1347 and erdafitinib, their clinical utility in patients with FGFR3 K650E mutant glioma is unknown.
```

Comment:
Seems like the summery under FGFR3/S371C, G380R, K650/Other Tumor Types is not updated by remove Debio1347. Probably good to check all descriptions whether `Debio1347` is still mentioned anywhere. May as well check `Infigratinib`, it was removed this May.

Reply:
What is the best way to remove `Debio1347` from description? Should I edit on the curation platform and ask for review, so it triggers a history save?

Reply:
@calvinlu3 could you look into the beta db and see which sections still have `Debio1347`? Then let the curation team know where they are, they should fix.